webcompat/webhooks/model.py

```diff
@@ -139,6 +144,14 @@ def prepare_public_comment(self):
         # prepare the payload
         return f'[Original issue {public_number}]({self.public_url})'
 
+    def reject_invalid_issue(self):
```

Comment:
Since there will be 3 of these, maybe it's cleaner to have a `reject_issue(reason=foo)`, where foo is `invalid`, `incomplete` or ...`rejected`). 

Or maybe `reject` is the wrong verb. 🤔 They're all getting closed... perhaps `close_public_issue(reason=foo`).